README.md

```diff
@@ -1 +1,21 @@
 # LFP Appeals Processor API
+
+## How to run
+### Run as a Spring Boot application
```

Comment:
```suggestion

### Run as a Spring Boot application
```

I believe adding extra link makes it more readable. I believe markdown files should be readable in raw format.